src/components/breadcrumb-header/index.tsx

```diff
@@ -3,23 +3,23 @@ import { compose } from 'redux';
 import { Breadcrumb } from '@fellesdatakatalog/breadcrumbs';
 import Link from '@fellesdatakatalog/link';
 
-import env from '../../env';
+import Translation from '../translation';
 
 import SC from './styled';
 
-const { FDK_BASE_URI } = env;
-
 const BreadcrumbHeader: FC = () => (
   <SC.BreadcrumbHeader>
     <SC.Breadcrumbs
       separator={<SC.BreadcrumbSeparator>{'>'}</SC.BreadcrumbSeparator>}
     >
       <Breadcrumb>
-        <Link href={`${FDK_BASE_URI}/publishing`} target='_self'>
-          Felles datakatalog publisering
+        <Link href='/' target='_self'>
```

Comment:
target='_self' er vel ikke nødvendig?

```suggestion
        <Link href='/'>
```

Reply:
Denne gjelder fortsatt